libs/@guardian/libs/src/measure/startMeasure.ts

```diff
@@ -31,16 +34,28 @@ export const startMeasure = (
 		start: performance.now(),
 	} satisfies PerformanceMeasureOptions;
 
-	const endMeasure = () => {
-		const measureName = serialise(team, name, action);
+	const measureName = serialise(team, name, action);
 
-		const { duration } = window.performance.measure(measureName, options) ??
+	const endMeasure = (end?: number) => {
+		const { duration } = window.performance.measure(measureName, {
+			...options,
+			end,
+		}) ??
```

Comment:
Just wondering when the LHS would this return nullish? According to the docs `measure` returns either a `PerformanceMeasure` or throws an exception.

Reply:
[Firefox before v101 returns `undefined`](https://github.com/guardian/csnx/pull/729/files#diff-d6ff5d412d9e790b174922bb617f2c51fc0df428a74cf235ab0df7c0b1dc6f49R4-R5)